Detections/DnsEvents/DNS_HighNXDomainCount_detection.yaml

```diff
@@ -1,8 +1,8 @@
 ﻿id: a0907abe-6925-4d90-af2b-c7e89dc201a6
-name: Rare high NXDomain count
+name: Potential DGA detected.
```

Comment:
removing the trailing period in name, we also may want to consider renaming the file to something like DNS_PotentialDGADetected.yaml.